cmake/version.cmake

```diff
@@ -46,9 +46,17 @@ function(DoGenerateVersionFile OUTPUT_FILE INPUT_FILE GIT_COMMIT_STATE_FILE
   string(REGEX MATCH "([0-9]+)\\." MAJOR_VERSION "${VERSION_STRING}")
   set(MAJOR_VERSION "${CMAKE_MATCH_1}")
 
+  if(NOT MAJOR_VERSION)
+    set(MAJOR_VERSION 0)
+  endif()
```

Comment:
Do we need a better replacement for this?

Reply:
Not sure. The one use case I currently have (fuzz tests) don't care at all about the version number.

Reply:
Discussed offline. Should be fine for now.